tests/drivers/rtc/rtc_api/doc/rtc_test.rst

```diff
@@ -0,0 +1,31 @@
+.. _rtc_api_test:
+
+:Author: Bjarki Arge Andreasen
+
+RTC Test suite
+###############
+
+Test RTC API implementation for RTC devices.
+
+Overview
+********
+
+This suite is design to be portable between boards. It uses the alias
+``rtc`` to designate the RTC device to test.
+
+This test suite tests the following:
+
+* Setting and getting the time.
+* RTC Time incrementing correctly.
+* Alarms if supported by hardware, with and without callback enabled
+* Calibration if supported by hardware.
+
+The calibration test tests a range of values which are printed to the
+console to be manually compared. The user must review the set and
+gotten values to ensure they are valid.
+
+By default, only the mandatory Setting and getting time is enabled
```

Comment:
```suggestion
By default, only the mandatory setting and getting of time is enabled
```

Reply:
Not sure why this is showing outdated? I opened all the commits just as I started reviewing this, github bug? If so/comment no longer applies, ignore